### ISSUE TITLE
buttons: Button height hardcoded as px

### ISSUE THREAD

**Issue by 2metres** [govau/uikit#171]
The height applied to buttons is being ignored when applying the button style to `<a>` or `<label>`.

<img width="249" alt="screen shot 2017-05-01 at 12 07 47 pm" src="https://cloud.githubusercontent.com/assets/924528/25570288/1482cf0c-2e67-11e7-8a84-6cbce54b590d.png">

HTML
```html
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>Title</title>
  <link rel="stylesheet" href="/assets/uikit.css">
</head>
<body class="uikit-body">
  <main class="uikit-grid">
    <div class="container">
      <p>
        <a class="uikit-btn" role="button" href="/start.html">Start</a>
      </p>
      <p>
        <a class="uikit-btn" role="button" href="#jump">Control Button</a> <!-- Jump to anchor -->
      </p>
     <div>
       <input hidden="" id="upload_reciept" type="file" role="button">
       <label class="uikit-btn uikit-btn--tertiary" for="upload_reciept">Upload reciept</label>
      </div>
    </div>
  </main>
</body>
```

**Comment by 2metres** [comment]
I've proposed a solution here: [#172](https://github.com/govau/uikit/pull/172)

However, if you're married to a fixed height for buttons you could always add explicit rules for `a.uikit-btn` and `label.uikit-btn`

**Comment by mgdhs** [comment]
@2metres I raised the same issue (differently) and got shot down. Nice to know someone else wants the same thing.

https://github.com/govau/uikit/issues/162

**Comment by dominikwilkowski** [comment]
I hope we don't shoot down people. I wand to encourage discussions. The problem is an accessibility one that I tried to explain in #162 .

Happy to keep the discussion open. We can repoen the old issue and discuss there to as it has the most infos so far.

**Comment by dominikwilkowski** [comment]
Buttons should not have a hardcoded px hight though.

**Comment by 2metres** [comment]
I fully agree with you regarding accessibility and semantics. I'm more looking for a way to represent an item visually as a button. 

Moving my comments to #162

**Comment by simonschwartz** [comment]
One possible approach we could take is to more defensively code our button component. 

This would mean that we don't recommend using a `.uikit-btn` class on an `<a>` element, hoever if a user decides to do this anyway we don't produce a broken looking component.

**Comment by simonschwartz** [comment]
Closing this issue. It has been fixed in #173
